umap/static/umap/js/modules/urls.mjs

```diff
@@ -0,0 +1,40 @@
+export default class URLs {
+    constructor(serverUrls) {
+        this.urls = serverUrls;
+    }
+
+    get(urlName, params) {
+        if (typeof params !== 'object')
+            params = { map_id: params };
```

Comment:
I guess you want to ensure that the `params` is an object with a `map_id` property.
`typeof []` also return 'object'.
`if(!params.map_id)` would seem more robust.
However this is not very clear to me and I may be misunderstanding